docs/openocr.md

```diff
@@ -80,6 +80,45 @@ tar xf OCR_e2e_img.tar
 python demo_gradio.py
 ```
 
+## Downloading Preprocessed Datasets
```

Comment:
These datasets were used to train the research version of SVTRv2, there are other data in the OpenOCR dataset, you can remove this data description for now, and we will subsequently add instructions for using the configuration file to download the dataset in SVTRv2.